packages/lib/server-only/attachment/set-template-attachments.ts

```diff
@@ -0,0 +1,80 @@
+import type { Attachment } from '@prisma/client';
+
+import { prisma } from '@documenso/prisma';
+
+import { AppError } from '../../errors/app-error';
+import { AppErrorCode } from '../../errors/app-error';
+
+export type CreateAttachmentsOptions = {
+  templateId: number;
+  attachments: Pick<Attachment, 'id' | 'label' | 'url' | 'type'>[];
+};
+
+export const setTemplateAttachments = async ({
+  templateId,
+  attachments,
+}: CreateAttachmentsOptions) => {
+  const template = await prisma.template.findUnique({
+    where: {
+      id: templateId,
```

Comment:
Need the team where query here otherwise you can update the attachment for any template with a crafted request.